phpunit.xml.dist

```diff
@@ -5,9 +5,6 @@
     <include>
       <directory suffix=".php">src</directory>
     </include>
-    <report>
```

Comment:
The presence of a `<report>` element when xdebug is disabled causes phpunit to grumble and paratest to fall over completely. Best to omit it and only enable reporting/coverage when it's needed.